routes/api/contacts.js

```diff
@@ -1,56 +1,132 @@
 const express = require("express");
-const contactsModel = require("../../models/contacts");
-const Joi = require("joi");
+const contactsRouter = express.Router();
+const Contact = require("../../models/contactModel");
+const { listContacts } = require("../../models/contacts");
 
-const router = express.Router();
+// const Joi = require("joi");
+// working with JSON file
 
-const contactSchema = Joi.object({
-  name: Joi.string().required(),
-  email: Joi.string().email().required(),
-  phone: Joi.string().required(),
-});
+// const contactSchema = Joi.object({
```

Comment:
You still should use `Joi` to validate payload for `post / patch / put` requests

Reply:
I use the contact schema in contactModel.js - with mongoose :)